mypy/checkexpr.py

```diff
@@ -204,7 +204,7 @@ def visit_call_expr(self, e: CallExpr, allow_none_return: bool = False) -> Type:
                         or isinstance(typ, NameExpr) and node and node.kind == nodes.TYPE_ALIAS):
                     self.msg.type_arguments_not_allowed(e)
         self.try_infer_partial_type(e)
-        callee_type = self.accept(e.callee)
+        callee_type = self.accept(e.callee, disallow_any=False)
```

Comment:
Did you mean updated in `typeshed`? 

Or silently treat them as `object` if this flag is enabled? I think the second is a good idea but would be really hard to implement, from a technical viewpoint.